app-backend/common/src/main/scala/utils/Shapefile.scala

```diff
@@ -2,18 +2,18 @@ package com.azavea.rf.common.utils
 
 object Shapefile {
 
+  @SuppressWarnings(Array("ListAppend"))
```

Comment:
Recommendation to use `cons` changes ordering. I don't know if a change like that would be acceptable.